components/sidebar/sidebar_channel/sidebar_direct_channel/sidebar_direct_channel.tsx

```diff
@@ -83,24 +85,6 @@ class SidebarDirectChannel extends React.PureComponent<Props, State> {
             return (
                 <i className='icon icon-archive-outline'/>
             );
```

Comment:
I suppose this condition could be reversed.
My suggestion would be something like this:

```
if (teammate?.id && teammate?.delete_at) {
    return <i className='icon icon-archive-outline'/>;
}

return null;
```

Reply:
we're returning the profile picture after so I can't flip it. Also I believe the early return is by Mattermost convention 🤔

Reply:
Fair enough ... I kinda missed, that the function does not get terminated after that bracket ... my bad 🙇🏻